CMakeLists.txt

```
@@ -33,6 +37,19 @@ set(CPACK_NSIS_DISPLAY_NAME ${APPLICATION_NAME})
 set(CPACK_NSIS_INSTALLED_ICON_NAME ${EXECUTABLE_NAME}.exe)
 set(CPACK_NSIS_EXECUTABLES_DIRECTORY .)
 set(CPACK_NSIS_MUI_FINISHPAGE_RUN ${EXECUTABLE_NAME}.exe)
+
+# Setup installer exe properties
+set (CPACK_NSIS_EXTRA_DEFINES 
+   VIProductVersion \"${VERSION}\"
+   VIFileVersion    \"${VERSION}.0\"
```

I believe it should be like this:
```suggestion
   VIProductVersion \"${CMAKE_PROJECT_VERSION}\"
   VIFileVersion    \"${CMAKE_PROJECT_VERSION}.0\"
```

---

OK. Thanks.